src/OrderTransfer/domain/order.go

```diff
@@ -0,0 +1,6 @@
+package domain
+
+type Order struct {
+	Id     uint16
+	Fields map[string]float64
```

Comment:
are we sure all values we get from an order ar float64? couldn't they be bool or enum?

Reply:
You're right, I'll make another commit